src/main/java/registrar/RegistrationController.java

```diff
@@ -0,0 +1,37 @@
+package registrar;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.apache.commons.validator.routines.EmailValidator;
+import javax.servlet.http.HttpServletResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Controller
```

Comment:
You probably need a @RestController annotation

Reply:
I don't do rest, actually I don't return anything, just redirect with is fine with Controller